tests/cross_functional/krkn_chaos/test_krkn_container_chaos.py

```
@@ -168,15 +163,13 @@ def test_krkn_container_kill_signals_chaos(
             )
             raise
         except Exception as e:
-            log.error(
-                f"❌ Multi-component container chaos with {kill_signal} failed: {e}"
+            logger.exception(
+                f"Multi-component container chaos with {kill_signal} failed: {e}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)